src/Reducer.js

```diff
@@ -56,18 +57,22 @@ function inject(state, action, props, scenes) {
     return state;
   }
   let ind;
+  let numPops = 1;
```

Comment:
Sorry, I didn't see yours before I create the PR #806 , I write something similar to your `numPops` . According this issue #39 , I think old version `Actions.pop()` could do `Actions.pop(2)` things, so I implemented this with several `assert` checking.